omp/test/matrix/sellp_kernels.cpp

```diff
@@ -243,4 +244,26 @@ TEST_F(Sellp, MoveToDenseIsEquivalentToRef)
 }
 
 
+TEST_F(Sellp, ExtractDiagonalIsEquivalentToRef)
+{
+    set_up_apply_data();
+
+    auto diag = mtx->extract_diagonal();
+    auto ddiag = dmtx->extract_diagonal();
+
+    GKO_ASSERT_MTX_NEAR(diag.get(), ddiag.get(), 0);
+}
+
+
+TEST_F(Sellp, ExtractDiagonalWithSliceSizeAndStrideFactorIsquivalentToRef)
```

Comment:
typo: (maybe also in the other tests?)
```suggestion
TEST_F(Sellp, ExtractDiagonalWithSliceSizeAndStrideFactorIsEquivalentToRef)
```